### PR TITLE
Add support for Amazon Linux 2023 (CVMFS 2.10.1)

### PR DESCRIPTION
**Pull request by ocaisa**

### CODE REVIEW

**Comment by ocaisa** [comment]
The CVMFS RHEL yum repo does not work on Amazon Linux, hence the CVMFS version in the file name.

File: scripts/install_cvmfs_2.10.1_eessi_AL2023.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```
Comment (trz42) [inline scripts/install_cvmfs_2.10.1_eessi_AL2023.sh:2]: Maybe add a header: "this file is part of ..., its purpose is to install CernVM-FS on Amazon Linux 2023 ... author: Alan O'Cais (@ocaisa) ... license"

**Review by trz42 — APPROVED** [review]
Looks good to me. Just recommended to add a header with a bit information.

File: scripts/install_cvmfs_2.10.1_eessi_AL2023.sh (hunk 1 above)
Comment (boegel) [reply scripts/install_cvmfs_2.10.1_eessi_AL2023.sh:2]: Yeah, it's not obvious that this is for Amazon Linux, I would have no idea what AL stands for.

Maybe script should also be renamed to `install_cvmfs_2.10.1_eessi_AmazonLinux-2023.sh`?

File: scripts/install_cvmfs_2.10.1_eessi_AL2023.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+sudo yum install -y http://ecsft.cern.ch/dist/cvmfs/cvmfs-config/cvmfs-config-default-latest.noarch.rpm
+sudo yum install -y https://ecsft.cern.ch/dist/cvmfs/cvmfs-2.10.1/cvmfs-2.10.1-1.el9.$(uname -m).rpm
+sudo yum install -y https://github.com/EESSI/filesystem-layer/releases/download/latest/cvmfs-config-eessi-latest.noarch.rpm
+
+sudo bash -c "echo 'CVMFS_CLIENT_PROFILE="single"' > /etc/cvmfs/default.local"
+sudo bash -c "echo 'CVMFS_QUOTA_LIMIT=10000' >> /etc/cvmfs/default.local"
+
+sudo service autofs start
```
Comment (bedroge) [inline scripts/install_cvmfs_2.10.1_eessi_AL2023.sh:10]: Was this really required? The next step `cvmfs_config setup` should automatically do this for you (see https://github.com/cvmfs/cvmfs/blob/devel/cvmfs/cvmfs_config#L304).

Comment (ocaisa) [reply scripts/install_cvmfs_2.10.1_eessi_AL2023.sh:10]: Hmm, probably not then, I will test and restart

File: scripts/install_cvmfs_2.10.1_eessi_AL2023.sh (hunk 1 above)
Comment (ocaisa) [reply scripts/install_cvmfs_2.10.1_eessi_AL2023.sh:2]: I actually had that and switched as that is the shorthand that they use, I can switch back in the next PR

**Pull request merged**